liquibase-standard/src/main/java/liquibase/change/core/SQLFileChange.java

```diff
@@ -161,4 +161,25 @@ public String describe() {
                 ", relativeToChangelogFile=" + relativeToChangelogFile +
                 '}';
     }
+
+    @Override
+    public CheckSum generateCheckSum() {
```

Comment:
The parent AbstractSQLChange class is already handing the checksum of the sql so a checksum of super.generateCheckSum plus doing this is double-computing?

Plus, I think we need excludeFromChecksum() to exclude the `path` an `relativeToChangeLogFile` properties in this file or the checksum will change if either of those change when it shouldn't be.

Reply:
We have decided to update this `generateChecksum()` implementation by not having to include any checksum part of the parent because for this particular change object, we only care about the checksum generated from the SQL file content. No need to exclude any field as we are already ignoring any of the SQL file change attributes.